src/types/ErrorType.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorType {
+  NoError = '',
```

Comment:
If there are no errors, just set the error state to `null`, no need to store extra enum